cmd/libs/go2idl/go-to-protobuf/protobuf/namer.go

```diff
@@ -167,6 +168,13 @@ func (n *protobufNamer) AssignTypesToPackages(c *generator.Context) error {
 			if t.Name.Package != p.PackagePath {
 				continue
 			}
+			// skip functions -- we don't care about them for protobuf
```

Comment:
actually, i guess you can leave them here, but there's a more fundamental filter that seems like it is missing..